test/runtests.jl

```diff
@@ -28,15 +28,15 @@ run(`make RUN=4263 CAMCOL=5 FIELD=119`)
 cd(wd)
 
 # Check whether to run time-consuming derivatives tests.
-test_derivatives_flag = "--test_derivatives"
-test_detailed_derivatives = test_derivatives_flag in ARGS
-test_files = setdiff(ARGS, [ test_derivatives_flag ])
+long_running_flag = "--long-running"
```

Comment:
@rgiordan -- FYI, I renamed `--test_derivatives` flag to `--long-running`. I thought it `--test_derivatives` was confusing because you can test some derivatives without that flag, and I thought `--long-running` could be used for other long running tests too.